backend/debugpy.py

```diff
@@ -0,0 +1,15 @@
+import os
+import debugpy
+import logging
+
+def config_to_bool(value: str) -> bool: 
+    return str(value).lower() in ('true', '1', 'yes', 'on')
+
+def check_and_enable_debugpy() -> None:
```

Comment:
It might be worth adding the additional configuration like we have in MyLA for this

https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/debug_utils/debugpy.py

And the default port is 5678, was there a reason 4030 was picked here?

We probably don't need the WAIT_FOR_ATTACH here, as that's mostly for debugging issue with the jobs in MyLA.

Reply:
Done. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/494/commits/fc9805201c523520c3765c1d4606db9481b5e153

I had to expose the default port in docker compose thought otherwise it wasn't stopping at breakpoint.